x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/40_unsupported_types.yml

```diff
@@ -391,6 +391,38 @@ unsupported with sort:
   - match: { values.0.27: "foo bar" }
   - match: { values.0.28: 3 }
 
+---
+spatial types with different precision in 8.12.x:
+  - skip:
+      version: " - 8.11.99, 8.13.0 - "
+      reason: "Elasticsearch 8.12 supported geo_point and cartesian_point with doc-values only and precision differences"
```

Comment:
should be `version: " - 8.11.99, 8.12.99 - "`?

Reply:
I believe the range is inclusive, not exclusive, so it is correct as it is. We skip everything earlier than 8.12.0 and everything from 8.13.0 and later. Basically this is the same as saying "test only 8.12.*".

However, looking at https://github.com/elastic/elasticsearch/pull/103947, it seems there is interest in removing all tests that try to test versions other than the current one. I suspect we might be asked to remove this test entirely.